From a0a0afc59bb1d2deddfdb776bc9e49e02f58ce32 Mon Sep 17 00:00:00 2001
From: "Kaleb S. KEITHLEY" <kkeithle@redhat.com>
Date: Tue, 24 Jul 2018 14:40:51 -0400
Subject: [PATCH] georep: fix hard-coded paths in gsyncd.conf.in

Why anyone puts a hard-coded path like /usr/local/sbin in a
config file is beyond me.

I wonder how many more are lurking in our sources

Change-Id: I6523894416cc06236ea1f99529efd36e957bd98e
updates: bz#1193929
Signed-off-by: Kaleb S. KEITHLEY <kkeithle@redhat.com>
---
 geo-replication/gsyncd.conf.in | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/geo-replication/gsyncd.conf.in b/geo-replication/gsyncd.conf.in
index 0a842cf2b..bd04400ff 100644
--- a/geo-replication/gsyncd.conf.in
+++ b/geo-replication/gsyncd.conf.in
@@ -138,11 +138,11 @@ value =
 help=If SSH keys are not secured with gsyncd prefix then use this configuration to set the actual path of gsyncd(Usually /usr/libexec/glusterfs/gsyncd)
 
 [gluster-command-dir]
-value=/usr/local/sbin/
+value=@SBIN_DIR@
 help=Directory where Gluster binaries exist on master
 
 [slave-gluster-command-dir]
-value=/usr/local/sbin/
+value=@SBIN_DIR@
 help=Directory where Gluster binaries exist on slave
 
 [gluster-params]
-- 
2.17.1

